lib/__tests__/_sendEvent.test.ts

```diff
@@ -31,8 +37,15 @@ describe("sendEvent", () => {
   });
 
   describe("with XMLHttpRequest", () => {
+    let AlgoliaInsights;
+    let sendBeaconBackup = window.navigator.sendBeacon;
```

Comment:
I guess you don't have to intialize it here if it's done in the beforeEach

Reply:
right. I've fixed! https://github.com/algolia/search-insights.js/pull/142/commits/faa50f7cd8420c8979547bedfe07fefa9dd6eec5